lib/ansible/modules/web_infrastructure/sentry_project.py

```diff
@@ -0,0 +1,166 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# (c) 2019, Mikhail Naletov <okgolove@markeloff.net>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = '''
+---
+module: sentry_project
+short_description: Manage Sentry project
+description:
+    - Manage the state of a project in Sentry.
+version_added: "2.8"
+
+options:
+  api_token:
+    description:
+      - API token generated in Sentry. The token sufficient access to manipulate projects.
+    required: true
+
+  organization:
+    description:
+      - Organization containing a project.
+    required: true
+
+  project_name:
+    description:
+      - Project name you want to work with.
+    required: true
+
+  project_slug:
+    description:
+      - Slug name. Slug name must be unique (even after project deletion you can't create a project with same slug name).
+    required: true
+
+  state:
+    description:
+      - The state of project.
+    required: true
+    choices: [ "present", "absent" ]
+
+  team:
+    description:
+      - The name of the team to create a new project for.
+
+  url:
+    description:
+      - Sentry URL.
+    required: false
+    default: sentry.io
+
+notes:
+  - Module supports check_mode, but it can't provide 100 percent gurantee what specified slug hasn't been used before.
+author: "Mikhail Naletov (@okgolove)"
+'''
+
+EXAMPLES = '''
+# Create project "backend" in "example" organization and "senior" team
+- sentry_project:
+    api_token: 1234567890abcdwxyz
+    organization: example
+    project_name: backend
+    project_slug: backend
+    state: present
+    team: senior
+    url: sentry.example.com
+'''
+
+RETURN = ''' # '''
+
+import json
+from ansible.module_utils.basic import AnsibleModule
+from ansible.module_utils.urls import fetch_url
+from ansible.module_utils._text import to_native
+from ansible.module_utils.six.moves.urllib.parse import urlparse
+
+
+def main():
+    arg_spec = dict(
+        api_token=dict(required=True),
```

Comment:
Same here: `type` should always be specified.
```suggestion
        api_token=dict(type='str', required=True),
```